### PR TITLE
Add new plan form

### PR DESCRIPTION
**Pull request by abdulwahidsharief**
### - Link added should be a valid URL and accessible throughout the org.
   ### - If the branch name contains hotfix / revert by default the BVT workflow check will pass.

| Test Case Document URL                        |
|-----------------------------------------------|
| Please paste test case document link here.... |

### CODE REVIEW

File: admin/controller/payment/razorpay.php
Comment (ramth05) [inline admin/controller/payment/razorpay.php:333]: @abdulwahidsharief , assign all empty variables at the start. no need to use else block.

File: admin/view/template/payment/razorpay_subscription/razorpay_plan_form.twig
Comment (ramth05) [inline admin/view/template/payment/razorpay_subscription/razorpay_plan_form.twig:166]: @abdulwahidsharief , is there any method to form the url? this looks like hardcoded and any changes in product may break it.

File: admin/model/payment/razorpay.php
Comment (ramth05) [inline admin/model/payment/razorpay.php:501]: @abdulwahidsharief , please create new variable for query and use proper line for each columns.

File: admin/view/template/payment/razorpay_subscription/razorpay_plan_form.twig
Comment (abdulwahidsharief) [reply admin/view/template/payment/razorpay_subscription/razorpay_plan_form.twig:166]: @ramth05 didn't find any method to get product catalog url

Comment (abdulwahidsharief) [reply admin/view/template/payment/razorpay_subscription/razorpay_plan_form.twig]: changed to catalog_product_url

**Review by ramth05 — APPROVED** [review]
LGTM, please test it after master merge thoroughly

**Pull request merged**
